.dockerignore

```diff
@@ -6,7 +6,7 @@ npm-debug.log
 *.env
 .env.*
 #  (uncomment to include in build for local dev env only -- do not distribute!)
-#!.development.env
+!.development.env
```

Comment:
README says that this line should be uncommented to make it work locally

Reply:
At runtime, not to be committed. README also says **DO NOT COMMIT SECRETS HERE**. Secrets management did not exist, so this was a first pass. Let's discuss options for secrets management that does not involve committing them to the repo.